WebRoot/js/zimbraMail/prefs/controller/ZmPrefController.js

```diff
@@ -349,6 +355,12 @@ function () {
 		}
 	}
 
+	var children = this._toolbar.getHtmlElement().children;
+	if(children && children[0]) {
+		children[0].setAttribute('role', 'region');
+		children[0].setAttribute('aria-label', ZmMsg.rightToolbar);
```

Comment:
can we use a variable for `children[0]`?

Reply:
Updated